pkg/detector/detector.go

```diff
@@ -271,11 +272,20 @@ func (d *ResourceDetector) EventFilter(obj interface{}) bool {
 		// The secret, with type 'kubernetes.io/service-account-token', is created along with `ServiceAccount` should be
 		// prevented from propagating.
 		// Refer to https://github.com/karmada-io/karmada/pull/1525#issuecomment-1091030659 for more details.
+		// The configmap and endpoints for leader election should be prevented from propagating.
 		case corev1.SchemeGroupVersion.WithKind("Secret"):
 			secretType, found, _ := unstructured.NestedString(unstructObj.Object, "type")
 			if found && secretType == string(corev1.SecretTypeServiceAccountToken) {
 				return false
 			}
+		case corev1.SchemeGroupVersion.WithKind("ConfigMap"):
+			if _, ok := unstructObj.GetAnnotations()[resourcelock.LeaderElectionRecordAnnotationKey]; ok {
+				return false
+			}
+		case corev1.SchemeGroupVersion.WithKind("Endpoints"):
+			if _, ok := unstructObj.GetAnnotations()[resourcelock.LeaderElectionRecordAnnotationKey]; ok {
```

Comment:
How about `Lease` resource?

Reply:
https://github.com/karmada-io/karmada/blob/7c96e0db541989bd95772bbc1cc0109d8479371c/pkg/util/apigroup.go#L54C3-L54C3

NewSkippedResourceConfig  disable Lease by default

Reply:
Thanks for reminding me.